doc/README~8.13~2021.02.md

```diff
@@ -248,7 +248,7 @@ The **full level** contains the following packages:
       (<a href='https://github.com/HoTT/HoTT/issues'>bug reports</a>)
       (<a href='https://coq.inria.fr/opam/released/packages/coq-hott/coq-hott.8.13/opam'>opam package</a>)
     </dd>
-    <dt><b>description</b></dt><dd>To use the HoTT library, the following flags must be passed to coqc:<br>   -noinit -indices-matter<br>To use the HoTT library in a project, add the following to _CoqProject:<br>   -arg -noinit<br>   -arg -indices-matter</dd>
+    <dt><b>description</b></dt><dd>To use the HoTT library, the following flags must be passed to coqc:<br>   -noinit -indices-matter<br>To use the HoTT library in a project, add the following to <code>_CoqProject</code>:<br>   -arg -noinit<br>   -arg -indices-matter</dd>
```

Comment:
Can such changes be removed? It somehow makes sense, but these files are auto generated from Opam metadata, and I don't want to get correction PRs each time i update the ReadMe files.